RoddyCore/src/de/dkfz/roddy/Roddy.java

```diff
@@ -589,8 +589,12 @@ private static void setDefaultRoddyJobIdVariable(RecursiveOverridableMapContaine
         configurationValues.add(new ConfigurationValue(CVALUE_PLACEHOLDER_RODDY_JOBID_RAW, "$" + Roddy.jobManager.getJobIdVariable()));
     }
 
+    private static void setDefaultRoddyJobNameVariable(RecursiveOverridableMapContainerForConfigurationValues configurationValues) {
```

Comment:
With all the (new) getters and setters, I think it might make sense to move all this to a custom class. Also we could have a RoddyStarter or so.. Might make things cleaner. Let's discuss it.

Reply:
Actually, I was also considering absorbing the single-line methods again into the only function they are called from. Generally, I think splitting the Roddy class may be useful, but I first want to understand which functions are there. The setters alone I would not consider enough for a class.